.gitattributes

```diff
@@ -1 +1,2 @@
 /testdata/** eol=lf
+# * text=auto
```

Comment:
Can we just delete this line, or add a comment above it to explain why it is commented out?